substrate/runtime-support/src/lib.rs

```diff
@@ -112,3 +112,38 @@ macro_rules! impl_outer_event {
 		)*
 	}
 }
+
+#[macro_export]
+macro_rules! impl_outer_log {
+	($(#[$attr:meta])* pub enum $name:ident for $trait:ident { $( $module:ident($( $log_type_name:ident => $log_type:ty => $as_log:ident ),*) ),* }) => {
```

Comment:
looks like there's always parens after the `$module:ident` (though potentially empty). prefer no parens to empty parens.

Reply:
Removed this at all, because of manual `DigestItem` impl